src/protocols/bdx/BdxTransferServerDelegate.h

```diff
@@ -46,7 +46,17 @@ class BDXTransferServerDelegate
      * @brief
      *  This method is invoked when a block is received.
      */
-    virtual CHIP_ERROR OnTransferData(BDXTransferProxy * transfer, const ByteSpan & data) = 0;
+    virtual CHIP_ERROR OnTransferData(BDXTransferProxy * transfer, const ByteSpan & data)
+    {
+        return CHIP_ERROR_NOT_IMPLEMENTED;
+    }
+
+    virtual CHIP_ERROR OnTransferData(BDXTransferProxy * transfer, const ByteSpan & data, bool isEof)
```

Comment:
Why is this needed?  The transfer ends OnTransferEnd, no?

Reply:
I understand. I will fix it.